website/versioned_docs/version-0.66/native-components-android.md

```diff
@@ -326,11 +353,22 @@ public class MyViewManager extends ViewGroupManager<FrameLayout> {
     }
   }
 
+  @ReactPropGroup(names = {"width", "height"}, customType = "Style")
+  public void setStyle(FrameLayout view, int index, Integer value) {
+    if (index == 0) {
+      propWidth = value;
+    }
+
+    if (index == 1) {
+      propHeight = value;
+    }
+  }
+
   /**
    * Replace your React Native view with a custom fragment
    */
   public void createFragment(FrameLayout root, int reactNativeViewId) {
-    ViewGroup parentView = (ViewGroup) root.findViewById(reactNativeViewId).getParent();
+    ViewGroup parentView = (ViewGroup) root.findViewById(reactNativeViewId);
```

Comment:
Was this intentional?

Reply:
I have backported the `unversioned` code to `0.66` which includes fixes from:
* #2843
> remove .getParent() because it was incorrectly left in by me last time

Docs for `0.66` included only applied the initial version of those changes, hence the update.

Reply:
Great thanks for clarifying